apps/etterlatte-tidshendelser/src/main/resources/logback-secure.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<included>
```

Comment:
Veit ikkje om vi treng sikkerlogg for denne appen, men viss vi gjer det bør den også setjast med properties i nais-yaml-filene og loggast til ved oppstart.

Reply:
Den trenger vel egentlig ikke det. Hovedgrunnen til at det ble lagt til er copy/paste av loggeropplegget til `fixedRateCancellableTimer` men ser at det ikke er påkrevd. Fjerner dette